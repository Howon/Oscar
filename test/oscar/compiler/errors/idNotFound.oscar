def main() => unit = {
  int x = y;
}