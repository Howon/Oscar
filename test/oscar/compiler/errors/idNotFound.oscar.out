Error: Identifier not found: y
